src/api/python/hidra/utils/utils_netgroup.py

```diff
@@ -0,0 +1,31 @@
+
+from ctypes import *
+
+def find_hostname(netgroup):
```

Comment:
I prefer `get` instead of `find` and the plural because multiple hostnames are returned:
```suggestion
def get_hostnames(netgroup):
```